kolibri/auth/models.py

```diff
@@ -26,120 +38,258 @@ class KolibriValidationError(KolibriError):
     pass
 
 
-class BaseUser(AbstractBaseUser):
+##############################################
+# MORANGO: These pieces will live in Morango, but dummies/stubs are temporarily included here until Morango is ready.
+##############################################
```

Comment:
Since we're constrained by this stuff, I'm uncomfortable not giving it any scrutiny. It opens the possibility that we'll code ourselves into a hole. Must it be included at this time *at all*? If so, can we at least we can decide on an API? In other words, can we decide the interface of Morango that this module can plug in to, in order to avoid tightly coupling the implementation?